api/src/commonMain/kotlin/Model.kt

```diff
@@ -297,15 +298,32 @@ data class TestCoverageDto(
     val name: String,
     val toRun: Boolean = false,
     val coverage: CoverDto = CoverDto(),
-    val stats: TestStats = TestStats(0, TestResult.PASSED),
+    val details: TestDetails = TestDetails(0, TestResult.PASSED),
 )
 
 @Serializable
-data class TestStats(
+data class TestDetails(
     val duration: Long,
     val result: TestResult,
+    val metadata: TestMetadata = TestMetadata.emptyMetadata,
 ) : JvmSerializable
 
+@Serializable
+data class TestMetadata(
+    val hash: String = "",
+    val data: Map<String, String> = emptyMap(),
+){
+    companion object {
+        val emptyMetadata = TestMetadata()
+    }
+}
```

Comment:
is a need to add JvmSerializable?

Reply:
we should remove JvmSerializable from every classes, because we are no longer use fst serialization